managed/src/main/java/com/yugabyte/yw/common/kms/util/EncryptionAtRestUtil.java

```diff
@@ -205,6 +207,25 @@ public static boolean configInUse(UUID configUUID) {
         return KmsHistory.configHasHistory(configUUID, KmsHistoryId.TargetType.UNIVERSE_KEY);
     }
 
+    public static ArrayNode getUniverses(UUID configUUID) {
+      Set<Universe> universes = KmsHistory.getUniverses(
+          configUUID, KmsHistoryId.TargetType.UNIVERSE_KEY);
+      ArrayNode details = Json.newArray();
+      
+      for (Universe universe : universes) {
+        UniverseDefinitionTaskParams universeDetails = universe.getUniverseDetails();
+        ObjectNode universePayload = Json.newObject();
```

Comment:
Declare this out side of  loop, we don't want to create multiple objects.

Reply:
we need to add the separate object to ArrayList, right?

Reply:
We can use the same ref to add into the ArrayList instead of creating new one everytime.

Reply:
yeah , got it. We can use it outside the for loop.